webui/react/Makefile

```diff
@@ -64,7 +64,7 @@ check-package-lock:
 	if grep 'ssh://' package-lock.json ; then echo "ssh url in package-lock.json, please convert to https url" ; false ; fi
 .PHONY: check-requirements
 check-requirements:
-	node --version | grep -E 'v16\.(1[3-9]|2[0-9])' || (echo "node version >=16.13 <17 is required" ; false)
+	node --version | grep -E 'v16\.(1[3-9]|2[0-9])|v17|v18|v19' || (echo "node version >=16.13 <20 is required" ; false)
```

Comment:
I think that long syntax was just a fancy way of saying newer than node 16.13 and up to 16.29 (inclusive). So we needed 16.13+ at some point but I'm not sure if we still do at this point. Up to you if you want to bump the version requirements up even higher.

Reply:
In Slack, Keita and I agreed to just be permissive now and revisit requirements when v20 (the next long-term support version) is available later this year